OpenRA.Mods.Common/Traits/World/SpawnMPUnits.cs

```diff
@@ -86,16 +86,18 @@ void SpawnUnitsForPlayer(World w, Player p, CPos sp)
 
 			foreach (var s in unitGroup.SupportActors)
 			{
-				var mi = w.Map.Rules.Actors[s.ToLowerInvariant()].TraitInfo<MobileInfo>();
-				var validCells = supportSpawnCells.Where(c => mi.CanEnterCell(w, null, c));
+				var actorRules = w.Map.Rules.Actors[s.ToLowerInvariant()];
+				var ip = actorRules.TraitInfo<IPositionableInfo>();
+				var validCells = supportSpawnCells.Where(c => ip.CanEnterCell(w, null, c));
 				if (!validCells.Any())
 				{
 					Log.Write("debug", "No cells available to spawn starting unit {0} for player {1}".F(s, p));
 					continue;
 				}
 
 				var cell = validCells.Random(w.SharedRandom);
-				var subCell = mi.SharesCell ? w.ActorMap.FreeSubCell(cell) : 0;
+				var ios = actorRules.TraitInfo<IOccupySpaceInfo>();
```

Comment:
We use the `Info` classes and `IPositionableInfo` doesn't inherit `IOccupiesSpaceInfo` afaik.